mayastor/src/grpc/v1/mayastor.rs

```diff
@@ -0,0 +1,306 @@
+use crate::{
+    bdev::{nexus, NvmeControllerState},
+    core::{BlockDeviceIoStats, CoreError, MayastorFeatures},
+    grpc::{
+        controller_grpc::{
+            controller_stats,
+            list_controllers,
+            NvmeControllerInfo,
+        },
+        rpc_submit,
+        GrpcClientContext,
+        GrpcResult,
+        Serializer,
+    },
+    host::{blk_device, resource},
+};
+use futures::FutureExt;
+use git_version::git_version;
+use rpc::mayastor::v1::mayastor as rpc;
+use std::panic::AssertUnwindSafe;
+use tonic::{Request, Response, Status};
+
+use ::function_name::named;
+
+/// RPC service for mayastor pool operations
+#[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
do we need this?

Reply:
will remove the dead code attribute.

Reply:
@tiagolobocastro We have the name field of the service declared, which is currently not used anywhere. Thats why dead code attribute is enabled.

Reply:
Done.